k9mail-library/src/main/java/com/fsck/k9/mail/internet/BinaryTempFileBody.java

```diff
@@ -137,12 +137,14 @@ public void close() throws IOException {
                 super.close();
             } finally {
                 Log.d(K9MailLib.LOG_TAG, "deleting temp file");
-                mFile.delete();
+                boolean fileDeleteResult = mFile.delete();
+                if (!fileDeleteResult)
+                    Log.i(K9MailLib.LOG_TAG, "fail to delete temp file");
```

Comment:
If you output a debug message please include the filename. Otherwise this message is almost useless.